dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunctionArguments.cs

```diff
@@ -13,6 +13,11 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public sealed class KernelFunctionArguments : Dictionary<string, string>
 {
+    /// <summary>
+    /// The main input parameter name.
+    /// </summary>
+    public const string InputParameterName = "input";
```

Comment:
We'll consider removing this subsequently?

Reply:
This will be addressed in scope of this issue - https://github.com/microsoft/semantic-kernel/issues/3761 shortly after the PR is merged.